spec/pubid_bsi/identifier_spec.rb

```diff
@@ -242,5 +242,33 @@ module Pubid::Bsi
 
       it_behaves_like "converts pubid to pubid"
     end
+
+    context "National Annexes" do
+      context "NA to BS EN 1999-1-2:2007" do
+        let(:pubid) { "NA to BS EN 1999-1-2:2007" }
+
+        it_behaves_like "converts pubid to pubid"
+      end
+
+      context "NA+A1:2012 to BS EN 1993-5:2007" do
+        let(:pubid) { "NA+A1:2012 to BS EN 1993-5:2007" }
+
+        it_behaves_like "converts pubid to pubid"
+      end
+
+      context "NA+A1:15 to BS EN 1993-1-4:2006+A1:2015" do
+        let(:original) { "NA+A1:15 to BS EN 1993-1-4:2006+A1:2015" }
+        let(:pubid) { "NA+A1:2015 to BS EN 1993-1-4:2006+A1:2015" }
```

Comment:
@mico I’d add that “NA+A1” means that the National Annex has an Amendment on the NA here. 

The document type here is a National Annex, not a BS. A National Annex is not a “suppplement” to the base document but can be treated as a separate document. Sometimes, the NA is also provided inside the document but we don’t know that from the identifier.